packages/preferences/src/browser/preferences-tree-widget.ts

```diff
@@ -469,16 +469,9 @@ export class PreferencesTreeWidget extends TreeWidget {
 
         this.id = PreferencesTreeWidget.ID;
 
-        this.properties = this.preferenceSchemaProvider.getCombinedSchema().properties;
-        for (const property in this.properties) {
-            if (property) {
-                // Compute preference group name and accept those which have the proper format.
-                const group: string = property.substring(0, property.indexOf('.'));
-                if (property.split('.').length > 1) {
-                    this.preferencesGroupNames.add(group);
-                }
-            }
-        }
+        this.toDispose.push(this.preferenceSchemaProvider.onDidPreferenceSchemaChanged(() => {
```

Comment:
Should not it go to `onAfterAttach`, i.e.:
```ts
this.initializeModel();
this.toDisposeOnDetach.push(this.preferenceSchemaProvider.onDidPreferenceSchemaChanged(() => {
    this.initializeModel();
}));
```

Reply:
Will try now. If it works, will update PR with this change. Thanks!

Reply:
Fixed

Reply:
Fixed